### ISSUE TITLE
Add huber loss

### ISSUE THREAD

**Issue by zorian15** [matsengrp/torchdms#139]
Last meeting we discussed giving RMSE &  [Huber Loss](https://en.wikipedia.org/wiki/Huber_loss) a go on CGG data. 

We currently don't support Huber loss in `torchdms`, but [PyTorch has an implementation](https://pytorch.org/docs/stable/generated/torch.nn.HuberLoss.html) that will be quick and easy to integrate.

I'll begin by just using the default parameters for delta, which results in a `SmoothL1Loss` -- but will add some capabilities to toggle this after things are up and running.
